src/SampleWebApp/Controllers/ControllerWithReadWriteActionsController.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleWebApp.Controllers
+{
+    public class ControllerWithReadWriteActionsController : Controller
```

Comment:
My bad, this was incorrectly created. 
For read write actions, it would generate all CRUD method stubs.